packages/web-components/src/components.d.ts

```diff
@@ -778,6 +778,14 @@ export namespace Components {
         "value"?: string;
     }
     interface VaSegmentedProgressBar {
+        /**
+          * Whether or not to center the step labels
+         */
+        "centeredLabels"?: boolean;
```

Comment:
Are all the news props v3 only? If so, lets make sure the description is updated to specifically mention that.